k2/csrc/host/arcsort.cc

```diff
@@ -21,15 +21,15 @@
 
 namespace k2 {
 void ArcSorter::GetSizes(Array2Size<int32_t> *fsa_size) const {
-  CHECK_NOTNULL(fsa_size);
+  K2_CHECK_NE(fsa_size, nullptr);
```

Comment:
it would be greate if we can define `K2_CHECK_NOTNULL`

Reply:
There was K2_CHECK_NOTNULL. Dan said it's the code that helps little but complex much. I finally get this principle. I used to write code w/o thinking that not to be too complex.

Reply:
Well, I disagree with @danpovey here..I likes the one `CHECK_NOTNULL`, it's more readable..

Reply:
I'm ok with CHECK_NOTNULL, but it doesn't take 20 lines of #ifdefs to check whether something is NULL.